### ISSUE TITLE
Add a dedicated page to the website for listing contributors

### ISSUE THREAD

**Issue by sappelhoff** [mne-tools/mne-bids#1271]
similar to what MNE-Python has at the bottom of their landing page, but perhaps more detailed?

xrefs:

- https://github.com/mne-tools/mne-python/issues/11605#issuecomment-2225073499

**Comment by cbrnr** [comment]
Yes! I think it (1) should be on a separate page (easier to cite if there is a dedicated URL, doesn't clutter the main page) and (2) should include more details, such as in https://github.com/all-contributors/all-contributors (they have roles which they display as emojis, maybe these roles can be consolidated/synced with roles in CITATION.cff).

**Comment by hoechenberger** [comment]
Great initiative
